src/pytest_docker_compose/__init__.py

```diff
@@ -186,4 +184,27 @@ def scoped_containers_fixture(docker_project: Project, request):
         return scoped_containers_fixture
 
 
+class ContainerDict(dict):
+    """A read-only dictionary that re-fetches containers from a docker-compose
+    project dynamically every time they are accessed."""
```

Comment:
❔ Could we include a reference to #8 and/or include some info about why this is necessary?  Good context for another developer coming across this and wondering why this class exists.

❔ Could we make the style match the rest of the code?  (triple quotes on their own line)

Reply:
yes, I agree. I will improve the documentation and stylize it more uniformly.

Reply:
Awesome, thanks!